noxfile.py

```diff
@@ -120,7 +127,7 @@ def run_scripts(session):
     # is fixed
     session.install("setuptools", silent=False)
     session.install("-e", ".[all,dev]", silent=False)
-    session.run("python", "run-tests.py", "--scripts")
+    session.run("python", "-m", "pytest", "tests/test_examples.py")
```

Comment:
```suggestion
    session.run("python", "-m", "pytest", "-v", "tests/test_examples.py")
```

Soon enough, we should enable more options here when this PR is closer to merging. Personally, I like to use `pytest -svra`.

Reply:
Do the options have any impact on the speed of the tests. I assume it would be negligible, but I am curious

Reply:
I too believe that i'd be negligible since there's no extensive logging but I guess we can definitely do some benchmarking before merging to quantify the range of time difference

Reply:
Yes, as Arjun says, the impact is indeed negligible and it isn't measurable either; `-v` is to increase the verbosity, `-s` controls any print statements or logging statements in stdout, and `-ra` will display a summary of everything: failed + skipped + xfailed/xpassed + errors, after the test suite completes.

Reply:
For reference, after we'll have most of this ready and when we'll be starting with the migration of the tests (which is from this week onwards), a useful guide with some tips is present at https://learn.scientific-python.org/development/guides/pytest/

Reply:
should i include `-svra` in all nox commands?

Reply:
Yes, should be okay

Reply:
Output with `-svra` is actually pretty clean. Thanks for the suggestion!